ansible/roles/hoover/tasks/main.yml

```diff
@@ -120,5 +120,26 @@
     dest: /opt/hoover/testdata
 
 - name: Create the testing_local.py config for snoop
-  template: src=hoover-snoop-config/testing_local.py
-            dest=/opt/hoover/snoop/snoop/site/settings/testing_local.py
+  copy: src=snoop-testing-local.py
+        dest=/opt/hoover/snoop/snoop/site/settings/testing_local.py
+
+- name: Create the local.py config for snoop
+  copy: src=snoop-local.py
+        dest=/opt/hoover/snoop/snoop/site/settings/local.py
+
+- name: Touch the secret_key.py config for snoop
+  file: dest=/opt/hoover/snoop/snoop/site/settings/secret_key.py
+        owner=liquid group=liquid state=touch
+
+- name: Create the local.py config for search
+  template: src=search-local.py
+            dest=/opt/hoover/search/hoover/site/settings/local.py
+            owner=liquid group=liquid
+
+- name: Touch the secret_key.py config for search
+  file: dest=/opt/hoover/search/hoover/site/settings/secret_key.py
+        owner=liquid group=liquid state=touch
+
+- name: Touch the oauth.py config for search
+  file: dest=/opt/hoover/search/hoover/site/settings/oauth.py
+        owner=liquid group=liquid state=touch
```

Comment:
Do you prefer the `x=y` syntax? Maybe we should use it everywhere then.

Reply:
I don't prefer it. Separate PR for #11.